Source/Extensions/UIStackViewExtension.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIStackViewExtension.swift
+//  Frames
+//
+//  Created by Ehab Alsharkawy
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+
+extension UIStackView {
+
+    func removeAllArrangedSubviews() {
+        let removedSubviews = arrangedSubviews.reduce([]) { (allSubviews, subview) -> [UIView] in
+            self.removeArrangedSubview(subview)
+            return allSubviews + [subview]
+        }
+        NSLayoutConstraint.deactivate(removedSubviews.flatMap({ $0.constraints }))
+        removedSubviews.forEach({ $0.removeFromSuperview() })
```

Comment:
Do we need to deactivate constraints? I would think they should automatically be removed?

I also found from UIKit description that:
```swift
    /* Removes a subview from the list of arranged subviews without removing it as
     a subview of the receiver.
        To remove the view as a subview, send it -removeFromSuperview as usual;
     the relevant UIStackView will remove it from its arrangedSubviews list
     automatically.
     */
    open func removeArrangedSubview(_ view: UIView)
```

With these in mind, I feel this could be optimised to:
```suggestion
    func removeSubviews() {
        subviews.forEach { $0.removeFromSuperview() }
    }
```

And then once its generic and functionality is not strictly related to UIStackView, we can move to UIViewExtensions file